src/lib/atoms/Button.svelte

```diff
@@ -1,89 +1,96 @@
 <script lang="ts">
   import type { HTMLAttributes } from 'svelte/elements';
-  import Typography from '../typography/Typography.svelte';
+  import Typography from '../design-system/Typography.svelte';
   import { cn } from '../utils/utils';
   import type { Snippet } from 'svelte';
+  import { cva, type VariantProps } from 'class-variance-authority';
 
-  interface Props extends HTMLAttributes<HTMLElement> {
+  const button = cva(
+    [
+      'h-[46px]',
+      'px-f24',
+      'cursor-pointer',
+      'rounded-full',
+      'text-center',
+      'transition-all',
+      'text-nowrap',
+      'hover:shadow-md',
+    ],
+    {
+      variants: {
+        intent: {
+          'filled-dark': ['bg-black', 'hover:bg-primary', 'text-white'],
+          'filled-light': ['bg-white', 'text-black'],
```

Comment:
The hover here only being the shadow disappears in places like the mobile nav menu:

<img width="612" alt="image" src="https://github.com/user-attachments/assets/87bacaa3-0a5b-406a-920a-8fb4150be5d5" />

However, it sounds like we are ok with this.

Reply:
I'll flag this with Heather later.